src/links/HttpLink.ts

```diff
@@ -0,0 +1,35 @@
+import { createApolloFetch } from 'apollo-fetch';
+import * as Observerable from 'zen-observable';
+
+import Link from '../';
+
+export default (uri: string) => {
+  // create HTTP utility
+  const fetcher = createApolloFetch({ uri });
+
+  // create Link from request and previous link
+  return new Link((request, prev = Link.empty()) => {
+    // hook up Observerable
+    return new Observerable(observer => {
```

Comment:
@evanshauser yeah I don't feel like we need to build an observable implementation? It feels like unnecessary work that we would have to maintain instead of relying on a project solely focused on it.

I like the size and narrow scope of Zen vs RxJSLite